package.json

```diff
@@ -47,7 +48,8 @@
     "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-terser": "^5.1.1"
+    "rollup-plugin-terser": "^5.1.1",
+    "prettier": "^1.18.2"
```

Comment:
Why we need prettier as a dev dependency?

Reply:
Maybe that is causing the formatting changes.

Reply:
it's possible that this could be used in different text editors like I am using Webstorm. So for common formatting, included prettier in devD

Reply:
Actually prettier is tied with eslint in vs code, I'm not sure how that will work with webstorm.
Can you check it otherwise the code will not be in sync?